fix-sync-issues.sh

```
@@ -0,0 +1,237 @@
+    with open('$CONFIG_FILE', 'r') as f:
+        config = yaml.safe_load(f)
+    print(config.get('dashboard', {}).get('port', 8766))
+except:
+    print('8766')
+" 2>/dev/null || echo "8766")
+else
+    DASHBOARD_PORT=8766
+fi
+
+# Matar procesos en puertos específicos
+ports=(5555 5556 55565 $DASHBOARD_PORT)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)